### ISSUE TITLE
ddb self-update permission error

### ISSUE THREAD

**Issue by Toilal** [inetum-orleans/docker-devbox-ddb#146]
`ddb self-update` seems to work only when command is runned with ddb directory (`~/.docker-devbox/bin`) as the working directory.

```
$ ddb self-update
A new version is available: 1.4.4
[core:selfupdate] An unexpected error has occured [phase:selfupdate => SelfUpdateAction.execute()]: You don't have permission to write on ddb binary file. (ddb)
```

**Comment by Toilal** [comment]
With 1.4.3

```
$ ddb -x self-update
A new version is available: 1.4.4
[core:selfupdate] An unexpected error has occured [phase:selfupdate => SelfUpdateAction.execute()]: You don't have permission to write on ddb binary file. (ddb)
Traceback (most recent call last):
  File "ddb/action/runner.py", line 81, in run
  File "ddb/action/runner.py", line 96, in _execute_action
```

**Comment by Toilal** [comment]
It still occurs in some situations, with a quite recent ddb version.

```
[core:selfupdate] An unexpected error has occured [phase:selfupdate => SelfUpdateAction.execute()]: You don't have permission to write on ddb binary file. (ddb)
Traceback (most recent call last):
  File "ddb/action/runner.py", line 81, in run
  File "ddb/action/runner.py", line 96, in _execute_action
  File "ddb/feature/core/actions.py", line 434, in execute
  File "ddb/feature/core/actions.py", line 447, in self_update_binary
PermissionError: You don't have permission to write on ddb binary file. (ddb)
dockeruser@ALM-ATOUT-vm:~/projects/ALM-ATOUT$ ~/.docker-devbox/bin/ddb self-update
A new version is available: 1.9.2
Downloading |████████████████████████████████| 100%
ddb has been updated.
```
